Classes/Hooks/ContentPostProcessorHook.php

```diff
@@ -123,7 +123,12 @@ protected function getConfiguration(array $rawConfiguration) {
     foreach ($configuration as $option => $value) {
 
       // Perform stdWrap processing on all options
-      if (StringUtility::isLastPartOfString($option, '.') && isset($value['stdWrap.'])) {
+      if (version_compare(TYPO3_version, '7.0', '>=')) {
```

Comment:
What's the benefit of using a variable instead of a low level PHP function for a simple compare? One could remove StringUtility::isLastPartOfString completely next month, because of TYPO3 6 LTS will be end of life.